tools/scripts/scvi/scvi-train.py

```diff
@@ -0,0 +1,73 @@
+import cellxgene_census
+import tiledbsoma as soma
+import yaml
+from cellxgene_census.experimental.pp import highly_variable_genes
+
+import scvi
+
+file = "scvi-config.yaml"
+
+with open(file) as f:
+    config = yaml.safe_load(f)
+
+census = cellxgene_census.open_soma()
```

Comment:
you will need to access the latest version for now to have access to `nnz`
```suggestion
census = cellxgene_census.open_soma(census_version="latest")
```